Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -3841,9 +3841,9 @@ WebIDL::ExceptionOr<void> Document::set_domain(String const& domain)
 
     // FIXME: 6. If the surrounding agent's agent cluster's is origin-keyed is true, then return.
```

Comment:
I wasnt sure about enabling the document domain setter without having this, but I suppose the previous same origin check was much more broken